vitest.config.js

```diff
@@ -33,18 +34,37 @@ const config = {
 
     exclude: [
       '**/builtin/**',
-      '**/node_modules/**',
-      '**/dist/**',
       '**/cypress/**',
-      '**/.{idea,git,cache,output,temp}/**',
+      '**/.{output,temp,cdix}/**',
```

Comment:
why removing the upper excludes ?

Reply:
they are included in configDefaults.exclude var, but I was playing around, it is better to have them extracted, will revert.

Reply:
ok thanks